examples/strings/strings.bal

```diff
@@ -22,14 +22,34 @@ public function main() {
     if (index is int) {
         io:println("IndexOf: ", index);
     }
-    
+
     // Retrieves the length of the `string`.
     int length = statement.length();
     io:println("Length: ", length);
 
+    string hello = "Hello";
+    string ballerina = "Ballerina!";
+
+    // Concats multiple strings.
+    string s4 = strings:concat(hello, " ", ballerina);
+    io:println(s4);
```

Comment:
Let's prefix what we print with the new examples with 
```
Xxx: 
```
similar to the rest of the examples.

The output would be uniform then.